### PR TITLE
Documentation: add more endpoints to flask #586

### PR DESCRIPTION
**Pull request by tbeerman**
Documentation: add more endpoints to flask #586

### CODE REVIEW

File: doc/source/restapi/archive.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Archive Rest Api
```
Comment (mlassnig) [inline doc/source/restapi/archive.rst]: API in all caps (also all files below ;-)

File: lib/rucio/web/rest/flaskapi/v1/archive.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+from logging import getLogger, StreamHandler, DEBUG
+from traceback import format_exc
+
+from flask import Flask, Blueprint, Response
+from flask.views import MethodView
+
+from rucio.api.did import list_archive_content
+from rucio.web.rest.flaskapi.v1.common import before_request, after_request
+
+LOGGER, SH = getLogger("rucio.meta"), StreamHandler()
```
Comment (mlassnig) [inline lib/rucio/web/rest/flaskapi/v1/archive.py:24]: rucio.meta ? shouldn't this be name of the module?

Comment (tbeerman) [reply lib/rucio/web/rest/flaskapi/v1/archive.py:24]: Yes, you're probably right. tbh I just copied the file from webpy, changed it to work with Flask and added the documentation. So, if there are any bugs or else I wouldn't have actually fixed them.

**Comment by vingar** [comment]
```
lib/rucio/web/rest/flaskapi/v1/temporary_did.py:14:1: F401 'flask.Response' imported but unused
lib/rucio/web/rest/flaskapi/v1/temporary_did.py:42:19: F821 undefined name 'generate_http_error'
lib/rucio/web/rest/flaskapi/v1/temporary_did.py:45:50: F821 undefined name 'ctx'
lib/rucio/web/rest/flaskapi/v1/temporary_did.py:50:20: W602 deprecated form of raising exception
```

**Review by vingar — APPROVED** [review]

**Pull request merged**